lua/entities/ent_jack_teslasentry/cl_init.lua

```diff
@@ -89,20 +89,17 @@ end
 local function ElectriTwitchClient( data )
     local Pos = data:ReadVector()
 
-    for key, rag in pairs( ents.FindInSphere( Pos, 40 ) ) do
-        if rag:GetClass() == "class C_ClientRagdoll" then
-            for i = 1, 60 do
-                timer.Simple( i / 20, function()
-                    if IsValid( rag ) then
-                        local Bones = rag:GetPhysicsObjectCount() - 1
-                        local Obj = rag:GetPhysicsObjectNum( math.random( 2, Bones ) )
-
-                        if Obj then
-                            Obj:ApplyForceCenter( VectorRand() * ( 60 - i ) * Obj:GetMass() * 10 )
-                        end
-                    end
-                end )
-            end
+    for _, rag in pairs( ents.FindInSphere( Pos, 40 ) ) do
+        if rag:GetClass() ~= "class C_ClientRagdoll" then continue end
+        for i = 1, 60 do
+            timer.Simple( i / 20, function()
+                if not IsValid( rag ) then return end
+                local Bones = rag:GetPhysicsObjectCount() - 1
+                local Obj = rag:GetPhysicsObjectNum( math.random( 2, Bones ) )
+
+                if not Obj then return end
+                Obj:ApplyForceCenter( VectorRand() * ( 60 - i ) * Obj:GetMass() * 10 )
+            end )

```

Comment:
Can we just remove this whole method for now? This is truly awful stuff (not your fault)
We'd also need to remove all of the usermessage stuff for `JackysElectriTwitchClientSentry` (both the sending and receiving)

Reply:
Applied